diff --git a/node_modules/vision-camera-code-scanner/android/build.gradle b/node_modules/vision-camera-code-scanner/android/build.gradle
index b6bd964..5904709 100644
--- a/node_modules/vision-camera-code-scanner/android/build.gradle
+++ b/node_modules/vision-camera-code-scanner/android/build.gradle
@@ -6,7 +6,7 @@ buildscript {
         }
 
         dependencies {
-            classpath 'com.android.tools.build:gradle:4.2.1'
+            classpath 'com.android.tools.build:gradle:7.4.1'
         }
     }
 }
@@ -18,11 +18,11 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('VisionCameraCodeScanner_compileSdkVersion', 30)
-    ndkVersion "21.4.7075529"
+    compileSdkVersion safeExtGet('VisionCameraCodeScanner_compileSdkVersion', 34)
+    ndkVersion "23.1.7779620"
     defaultConfig {
-        minSdkVersion safeExtGet('VisionCameraCodeScanner_minSdkVersion', 21)
-        targetSdkVersion safeExtGet('VisionCameraCodeScanner_targetSdkVersion', 31)
+        minSdkVersion safeExtGet('VisionCameraCodeScanner_minSdkVersion', 23)
+        targetSdkVersion safeExtGet('VisionCameraCodeScanner_targetSdkVersion', 34)
         versionCode 1
         versionName "1.0"
 
